readme.md

```diff
@@ -157,6 +157,7 @@ Thanks for contributing! 🦋🙌
 - [](# "esc-to-deselect-line") [Adds a keyboard shortcut to deselect the current line: <kbd>esc</kbd>.](https://github.com/sindresorhus/refined-github/issues/1590)
 - [](# "html-preview-link") [Adds a link to preview HTML files.](https://user-images.githubusercontent.com/44045911/67634792-48995980-f8fb-11e9-8b6a-7b57d5b12a2f.png)
 - [](# "vertical-front-matter") [Show Markdown front matter as vertical table.](https://user-images.githubusercontent.com/44045911/87251695-26069b00-c4a0-11ea-9077-53ce366490ed.png)
+- [](# "follow-local-link") [Stay on the local page while jumping to the definition of a function or method within the same file.](https://user-images.githubusercontent.com/16872793/90635640-7091da80-e1f7-11ea-9d79-eb340f9d2c61.png)
```

Comment:
Maybe 
```suggestion
- [](# "same-page-definition-jump") [Stay on the local page while jumping to the definition of a function or method within the same file.](https://user-images.githubusercontent.com/16872793/90635640-7091da80-e1f7-11ea-9d79-eb340f9d2c61.png)
```

Also needs to be moved to the GitHub bugs section

Reply:
The screenshot needs to be updated with something more realistic, not a class that points to itself; and without text

Reply:
<img width="904" alt="" src="https://user-images.githubusercontent.com/1402241/90641702-70aace00-e229-11ea-946c-3a76697b9184.png">

Example https://github.com/sindresorhus/refined-github/blob/master/source/github-helpers/index.ts#L11

Reply:
I should use that screenshot?

Reply:
I think you could add an arrow going from the 46 in the popup to the 46 on the left, and add the text "it instantly highlights line 46 on click instead of reloading the page" under the popup

Reply:
![image](https://user-images.githubusercontent.com/16872793/90833649-7a5e2f80-e316-11ea-827d-a4e3ac8ced69.png)

Good?